pages/common/gh-issue.md

```diff
@@ -0,0 +1,28 @@
+# gh-pr
+
+> Manage GitHub issues from the command line.
+> More information: <https://cli.github.com/manual/gh_issue>.
+
+- Print out the issue:
+
+`gh issue view {{issue_number}}`
+
+- Create a new issue in the web browser:
```

Comment:
I prefer to get emphasis to what the command does, and not what the user does in the browser itself.
```suggestion
- Open a browser window for create a new issue:
```

Reply:
Here, again, I wanted to focus more on "what user wants to do", not what technically command does

Reply:
Hrm, I see your point @TheLastGimbus. I don't think it really matters either way here - I think I actually prefer @TheLastGimbus' original example. They both have merit though, since @navarroaxel's highlights the fact that a new web browser window is opened to create the issue, rather than the browser window being opened in